pkg/runtime/wfengine/state/state.go

```diff
@@ -116,6 +116,11 @@ func (s *State) AddToInbox(e *backend.HistoryEvent) {
 	s.inboxAddedCount++
 }
 
+func (s *State) AddToHistory(e *backend.HistoryEvent) {
```

Comment:
Is this an intentional add? I dont see any tests for it or references in this PR other than the diff here.

Reply:
Thanks, good catch. Removed.